tests/command_line/test_export.py

```diff
@@ -584,6 +584,39 @@ def test_shelx_ins_best_unit_cell(dials_data, tmp_path):
                 assert result == pytest.approx(cell_esds[instruction], abs=0.001)
 
 
+def test_shelx_ins_composition(dials_data, tmp_path):
```

Comment:
Is it worth adding a test which actually executes `shelxt` if available to verify?